combat-trainer.lic

```diff
@@ -352,6 +355,12 @@ class LootProcess
     end
   end
 
+  def should_perform_ritual(game_state)
+    unless game_state.prepare_consume
```

Comment:
I think `game_state.casting_consume` also should be considered, as when thats true youre waiting for the heal spell to become fully prepped. maybe `unless game_state.prepare_consume || game_state.casting_consume`